coalib/bears/requirements/PackageRequirement.py

```diff
@@ -49,3 +49,38 @@ def check(self):
         :return: Returns True if satisfied, False if not.
         """
         raise NotImplementedError
+
+    @classmethod
+    def multiple(cls, *args):
```

Comment:
@Adrianzatreanu What @sils1297 is saying (I think) is that if you have let's say a class like `Bear` ... this is a template right ? It has no features but does boiler plate code for all other bears. So, if a user were to create an object of Bear `b = Bear()` it is actually not valid, as it's just meant as a template.

Now, this classmethod provides a way to create an object of an abstract class, which is non-intuitive, and confusing.
i.e. `PackageRequirement.multiple(('pip', 'coala_decorators', '0.1.0'))` can create an object of `PackageRequirement`.

There are many possible solutions (assuming I understood the issue @sils1297 is trying to tell us):
 - I personally don't mind this. It's hacky, but it works. And if a user does try to create an object of a class we say is abstract in the docs, they should expect unusual outputs.
 - You can check that `cls` is not this class itself, but is a derived class. This can be done using the `issubclass` builtin function or even with a simple `if cls == PackageRequirement:` -> `raise NotImplementedError`
 - Finally, python has an "abstract base class" module - https://docs.python.org/3/library/abc.html I've never used it myself, but it's the official way to do it. It has something called `abstractclassmethod` ... but I don't think it's what you want here.

2nd option seems best to me.